tests/tests_data/test_import_time.py

```diff
@@ -0,0 +1,20 @@
+import os
+import sys
+from time import time
+
+import pytest
+
+
+@pytest.mark.skipif(sys.platform == "win32", reason="windows isn't supported")
+def test_lightning_data_import():
+    t0 = time()
+    code = "import lightning; assert 'LightningModule' in dir(lightning)"
+    assert not os.system(f'python -c "{code}"')
+    t1 = time()
+
+    t2 = time()
+    code = "import lightning; assert 'LightningModule' not in dir(lightning)"
+    assert not os.system(f'LIGHTNING_LAZY_IMPORTS=1 python -c "{code}"')
+    t3 = time()
+
+    assert (t1 - t0) > (t3 - t2) + 1
```

Comment:
Why the +1?

Reply:
It is a good difference to measure. It could possibly be more.